src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/tests/test_rdbms_commands.py

```diff
@@ -39,15 +40,18 @@ def __init__(self, engine_type='mysql', engine_parameter_name='database_engine',
         self.admin_password = admin_password
         self.resource_group_parameter_name = resource_group_parameter_name
         self.skip_delete = skip_delete
+        self.sku_name = sku_name
+        
 
     def create_resource(self, name, **kwargs):
         group = self._get_resource_group(**kwargs)
-        template = 'az {} server create -l {} -g {} -n {} -u {} -p {}'
+        template = 'az {} server create -l {} -g {} -n {} -u {} -p {} --sku-name {}'
```

Comment:
We should support create server with only required parameters (location/resourcegroup/name/adminUser/adminPassword), sku is optional.  Please remove --sku-name here.